Classes/ContentObject/FloatContentObject.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the "headless" Extension for TYPO3 CMS.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.md file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace FriendsOfTYPO3\Headless\ContentObject;
+
+use Psr\Log\LoggerAwareTrait;
+use TYPO3\CMS\Frontend\ContentObject\AbstractContentObject;
+
+/**
+ * FLOAT Content Object only for use in JSON content object
+ *
+ * ** not working ** outside of JSON content object
+ */
+class FloatContentObject extends AbstractContentObject
+{
+    use LoggerAwareTrait;
```

Comment:
what do we need this trait for?

Reply:
Not needed, but it is just a copy of the INT cObject. Perhaps we check this in all classes after?!